src/overlays/actors/ovl_Demo_Getitem/z_demo_getitem.c

```diff
@@ -62,22 +69,23 @@ void DemoGetitem_Init(Actor* thisx, PlayState* play) {
 void DemoGetitem_Destroy(Actor* thisx, PlayState* play) {
 }
 
-void func_80A4FB10(DemoGetitem* this, PlayState* play) {
+void DemoGetitem_Wait(DemoGetitem* this, PlayState* play) {
     if (Object_IsLoaded(&play->objectCtx, this->objectIndex)) {
         this->actor.draw = NULL;
         this->actor.objBankIndex = this->objectIndex;
-        this->actionFunc = func_80A4FB68;
+        this->actionFunc = DemoGetitem_PerformCutsceneActions;
     }
 }
 
-void func_80A4FB68(DemoGetitem* this, PlayState* play2) {
-    PlayState* play = play2;
-    u16 sp22 = (play->gameplayFrames * 1000) & 0xFFFF;
+void DemoGetitem_PerformCutsceneActions(DemoGetitem* this, PlayState* play) {
+    s32 pad;
+    u16 bobPhase = (play->gameplayFrames * 1000) & 0xFFFF;
```

Comment:
```suggestion
    u16 bobPhase = (play->gameplayFrames * 1000) & 0xFFFF;
```
why not robertPhase? 😆